labgrid/remote/client.py

```diff
@@ -922,7 +922,11 @@ def create():
     _, host, port, _, _, _ = parse_url(url)
 
     coro = loop.create_connection(transport_factory, host, port)
-    loop.run_until_complete(coro)
+    try:
+        loop.run_until_complete(coro)
+    except ConnectionError as e:
+        print("Could not connect to coordinator: {}".format(e))
+        exit(1)
```

Comment:
This is part of the library layer, so we can't just exit(1) here. For labgrid-client you can move the try/except to the start_session call site in main(). RemotePlaceManager already has a similar try/except.

Reply:
Fixed as suggested.